src/modules/navigator/navigation.h

```diff
@@ -48,6 +48,8 @@
 #  define NUM_MISSIONS_SUPPORTED 50
 #elif defined(__PX4_POSIX)
 #  define NUM_MISSIONS_SUPPORTED (UINT16_MAX-1) // This is allocated as needed.
+#elif defined(RAM_BASED_MISSIONS)
+#  define NUM_MISSIONS_SUPPORTED 1000
```

Comment:
Did you mean to keep this (1000 vs 500)?

Reply:
No opinion. I guess I chose 1000 because it was half 2000. I'll change it to 500.